webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -1,25 +1,73 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {getSuggestionsSplitBy, getSuggestionsSplitByMultiple} from 'mattermost-redux/utils/user_utils';
-import {makeGetProfilesInChannel} from 'mattermost-redux/selectors/entities/users';
-import {makeAddLastViewAtToProfiles} from 'mattermost-redux/selectors/entities/utils';
+import { ActionResult } from 'mattermost-redux/types/actions';
+import { getSuggestionsSplitBy, getSuggestionsSplitByMultiple } from 'mattermost-redux/utils/user_utils';
+import {Group} from '@mattermost/types/groups';
+import { makeGetProfilesInChannel } from 'mattermost-redux/selectors/entities/users';
+import { makeAddLastViewAtToProfiles } from 'mattermost-redux/selectors/entities/utils';
+import {UserProfile, UserProfileWithLastViewAt} from '@mattermost/types/users';
 
 import store from 'stores/redux_store';
-import {Constants} from 'utils/constants';
+import { Constants } from 'utils/constants';
+import { GlobalState } from 'types/store';
 
 import Provider from '../provider';
 
 import AtMentionSuggestion from './at_mention_suggestion';
 
-const profilesInChannelOptions = {active: true};
+const profilesInChannelOptions = { active: true };
```

Comment:
Avoid change small code and reorder import

Reply:
Yeah, I will reorder that. And I guess I have to turn off my prettier extension. Because of that, these small changes happened.